src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -213,7 +213,10 @@ export const ProfilePage = () => {
                   <div className={assetList}>
                     {ownerAssets?.length
                       ? ownerAssets.map((asset, index) => (
-                          <WalletAssetDisplay asset={asset} isSellMode={isSellMode} key={index} />
+                          // eslint-disable-next-line react/jsx-key
```

Comment:
for formatting purposes for now, I will fix this when I do the aspect ratio / nft image sizing ticket

Reply:
Is the issue the wrapping div not having a key? You might be able to just move the `key={index}` up one level